appscale/tools/parse_args.py

```diff
@@ -361,6 +361,13 @@ def add_allowed_flags(self, function):
         help="the name of the AppController property to set")
       self.parser.add_argument('--property_value',
         help="the value of the AppController property to set")
+    elif function == "appscale-upgrade":
+      self.parser.add_argument('--keyname', '-k', default=self.DEFAULT_KEYNAME,
+        help="the keypair name to use")
+      self.parser.add_argument('--ips_layout',
+        help="a YAML file dictating the placement strategy")
+      self.parser.add_argument('--unique_ips',
```

Comment:
For running the bootstrap on all machines, I needed the unique ips and there was an already existing method here to parse that from ips layout yaml. On the appscale-tools, I would have to parse it from a string, so I thought it would be better to pass it here.